site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md

```diff
@@ -0,0 +1,364 @@
+
+Now, time to jump back to the Snowflake configuration tab.
+
+### Now Let's Shake
+Duration: 1
+
+1. Paste the  **Client id** from ServiceNow into the Snowflake configure pop-up.
+1. Go back to the ServiceNow tab and copy the **Client secret** and paste it into the Snowflake configure pop-up. 
+1. No need to change the Advanced Settings, but feel free to check them out.
+ ![Connect](assets/now_connect.png)
+1. Select **Connect**. Your ServiceNow accounts pops up and requests to connect to Snowflake. 
+![check](assets/now_check.png)
+1. Select **Allow**.
+The connection is established between the two systems. 
+
+To verify the connection, select the three dots [...] and **View Details**. At the top of the pop-up you will see the date **ServiceNow** Authenticated.
+
+![authenticated](assets/authenticated.png)
+
+Select **Done**. 
+
+> aside negative
+> If you are having issues, perhaps the password wasn't copied. Unlock the password field and copy and paste the text.
+## Configure the Connector
+Duration: 5
+
+Under the status for the connector, which displays "Choose Resources", select **Configure**.
+
+This displays the Configure Connector dialog. By default, the fields are set to the names of objects that are created when you configure the connector.
+
+![default config](assets/configuredefaults.png)
+
+Check out [Configuring the Snowflake Connector for ServiceNow](https://other-docs.snowflake.com/en/connectors/servicenow/servicenow-installing-ui.html#configuring-the-snowflake-connector-for-servicenow) for more information on these fields. 
+
+Select **Configure**. The dialog box closes and the status of the connector changes to Provisioning. It can take a few minutes for the configuration process to complete.
+
+> aside negative
+> Watch out!!! The created warehouse is created as a **Large** and with a auto timeout of 10 minutes. So this means, if you set to refresh every hour, the Large warehouse (8 credits/hour) will wake up for a minimum of 10 minutes every hour.  For this lab, you don't need all the power! Go to Admin-> Warehouses -> SERVICENOW_WAREHOUSE -> ... > Edit, and change this to an XSMALL, and the auto timeout to one minute.
+
+> aside positive
+> Absolutely attach a resource monitor to the SERVICENOW_WAREHOUSE. Go to Admin->Resource Monitors->+ Resource Monitor, and create a warehouse resource monitor:
+![resource monitor](assets/monitor.png)
+
+## Select ServiceNow Tables
+Duration: 4
+
+> aside negative
+> A couple of things to be aware of:
+> - The connector can only ingest tables with **sys_id** columns present.
+> - ServiceNow views are not supported. Instead of ingesting these views, you should synchronize all tables for the underlying view and join the synchronized tables in Snowflake.
+> - Incremental updates occur only for tables with **sys_updated_on** or **sys_created_on** columns.
+> - For tables that do not have sys_updated_on or sys_created_on columns, the connector uses **truncate and load** mode.
```

Comment:
WDYT about adding some additional explanation what does **truncate and load** mode mean?

> In this mode, the table is always ingested using the initial load approach, and newly ingested data replaces the old data.